bin/stringfuzzx

```diff
@@ -43,13 +43,14 @@ DEFAULT_INTEGER_FLAG  = False
 DEFAULT_SKIP_RE_RANGE = True
 
 GET_MODEL = "get-model"
-SET_INFO  = "set-info"
 GET_INFO  = "get-info"
-TO_STRIP  = [GET_MODEL, SET_INFO, GET_INFO]
+TO_STRIP  = [GET_MODEL, GET_INFO]
 
-def is_not_suppressed(expr):
+def should_keep(expr):
     if isinstance(expr, SettingNode):
         return False
+    if isinstance(expr, MetaExpressionNode):
+        return False
     if isinstance(expr, ExpressionNode):
```

Comment:
Just to confirm, get-model and get-info are still ExpressionNodes?

Reply:
Yes.

Reply:
I can make them be something else, but I won't. After the paper deadline, I'll replace the whole parser with ANTLR, because my own parser-writing skills are not up to par.